addons/zylann.hterrain/util/util.gd

```diff
@@ -57,6 +57,41 @@ static func create_wirecube_mesh(color = Color(1,1,1)) -> Mesh:
 	return mesh
 
 
+static func file_get_23(f: FileAccess) -> int:
```

Comment:
Why is this `get_23`? We are getting 3 bytes, it should be `get_24`...

Reply:
Thanks for catching that, that was just a brain derp